specification/sql/resource-manager/Microsoft.Sql/preview/2023-02-01-preview/ManagedDatabaseSensitivityLabels.json

```diff
@@ -854,6 +854,19 @@
             "name": "SensitivityLabelRank",
             "modelAsString": false
           }
+        },
```

Comment:
can you please revert Venkat's change? when your PR is merged, it would auto rebase or sync.

Reply:
done